src/assets/translations/en/main.json

```diff
@@ -18,6 +18,7 @@
     "relayerGasFee": "Relayer Gas Fee",
     "insufficientAmountForGas": "Not enough ETH to cover gas",
     "invalidAddress": "Invalid Address",
+    "invalidAddressOrYat": "Invalid Address or Yat",
```

Comment:
please revert all of these translations to the other languages, other than this english file.